launch/cylinder_spawner.launch

```diff
@@ -0,0 +1,5 @@
+<launch>
+<!-- Spawn a robot into Gazebo -->
+    <param name="cylinder_description" command="$(find xacro)/xacro.py $(find vito_description)/model/cylinder.urdf.xacro" />
```

Comment:
But I missed this... the force-torque-sensor package shouldn't depend on vito. If you need the cylinder to place it on top of the sensor, copy it into the force-torque-sensor package.